copyist.go

```diff
@@ -86,7 +86,7 @@ var registered *proxyDriver
 // testing utility code wants to automatically determine whether to open a
 // connection using the copyist driver or the "real" driver.
 func IsOpen() bool {
-	return registered.recording != nil
+	return registered != nil && registered.recording != nil
```

Comment:
NIT: Can you add a check for this in `TestOpenWithoutRegister` - just a `require` check that calling `IsOpen` doesn't panic.

Reply:
Done.